File: Easy/Add'em Up/Add'em Up.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class Solution
{
    public static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        List<int> pointvalue = Console.ReadLine().Split().Select(x => int.Parse(x)).ToList();
        
        int totalcost = 0;
        
        while (pointvalue.Count() > 1)
        {
            pointvalue.Sort();
            int minSum = pointvalue.ElementAt(0) + pointvalue.ElementAt(1);
            
            totalcost += minSum;

            pointvalue.RemoveAt(0);
            pointvalue.RemoveAt(0);
            pointvalue.Add(minSum);
        }
    Console.WriteLine(totalcost);
    }
}

File: Easy/Add'em Up/Add'em Up.py
n = int(input())
a = list(map(int,input().strip().split()))[:n]
totalcost = 0
        
while len(a) > 1:
    a.sort()
    sum = a[0] + a[1]
    totalcost += sum
    del a[0]
    del a[0]
    a.append(sum)

print(totalcost)